### PR TITLE
Fix #3 issue

### LINKED ISSUE #3: Make public oracle libraries to avoid crash build
Oracle instant client and jdk are in Viva private repos.
We have to change this libraries to public repo.

### PR DESCRIPTION
**Pull request by wilo087**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11,6 +11,22 @@ Clone the repo
 git clone git@gitlab.viva.com.do:anaiboa/pl-admin.git
```
Comment (AdrianMendez1199) [inline docs/README.md]: change the url to the github repo

Comment (wilo087) [reply docs/README.md]: Ready!

**Review by AdrianMendez1199 — COMMENTED** [review]
god job

**Review by AdrianMendez1199 — APPROVED** [review]

**Review by wilo087 — COMMENTED** [review]
I think that everything it's ok

**Pull request merged**